dev/devicelab/manifest.yaml

```diff
@@ -124,6 +124,13 @@ tasks:
     stage: devicelab
     required_agent_capabilities: ["mac/android"]
 
+  run_release_test:
+    description: >
+      Checks that `flutter run --release` does not crash.
+    stage: devicelab
+    required_agent_capabilities: ["mac/android"]
```

Comment:
Seems like it'd be good to have this on iOS too

Reply:
Though the test is pretty Android-centric in its expectations of the output...

Reply:
Yeah. Let's start with an Android test and add the iOS test separately if it makes sense.